projects/observability/src/shared/graphql/model/metrics/metric-aggregation.ts

```diff
@@ -94,7 +94,7 @@ export const addAggregationToDisplayName = (displayName: string, aggregation: Me
     case MetricAggregationType.P95:
     case MetricAggregationType.P90:
     case MetricAggregationType.P50:
-      return displayName;
+      return `${displayName} ${getAggregationDisplayName(aggregation)}`;
```

Comment:
Can we use the return statement for the `Min`, `Max`, `Count`, `Sum` for the `P99` and others?
It currently returns `${getAggregationDisplayName(aggregation)} ${displayName}`, which should work for our case too.

Reply:
For other cases also `postfix` and `prefix` aggregations are present. Please check the entire function.

Reply:
@jaywalker21 made the change for `prefix` aggregation for avg, p99, p95, p90, p50, min, max, sum. Please check.